redis_benchmarks_specification/__self_contained_coordinator__/self_contained_coordinator.py

```diff
@@ -912,9 +912,7 @@ def process_self_contained_coordinator_stream(
                                     logging.critical("Printing redis container log....")
                                     print("-" * 60)
                                     print(
-                                        redis_container.logs(
-                                            stdout=True, stderr=True, logs=True
```

Comment:
Also there was an issue with this line in contained_coordinator. When benchmark raised an exception for any reason we tried to print container logs but it resulted with: `TypeError: ContainerApiMixin.logs() got an unexpected keyword argument 'logs'`. According to https://docker-py.readthedocs.io/en/stable/containers.html#docker.models.containers.Container.logs parameter `logs `for method `logs()` doesn't exist. This led to skipping 'teardown' stage which led to temporary folders being left with the redis-server artifact inside.

Reply:
With that change instead we get:
```
2023-09-13 01:33:49 CRITICAL Some unexpected exception was caught during local work. Failing test....
2023-09-13 01:33:49 CRITICAL <class 'KeyError'>
2023-09-13 01:33:49 INFO Tearing down setup
2023-09-13 01:33:49 INFO When trying to stop DB container with id a9b45d05a8ba306464d79d3d0deeaa2a72dd7d8761b838aefe5d318326e1cf5e and image <Image: 'debian:buster'> it was already stopped
2023-09-13 01:33:49 INFO Removing temporary dirs /root/tmp7rhog4yu and /root/tmpkcu7fzqu
```
And temporary dirs are removed